packages/pharos/src/components/popover/pharos-popover.scss

```diff
@@ -0,0 +1,44 @@
+@use '../../utils/scss/mixins';
+
+:host {
+  display: block;
+  position: fixed;
+  z-index: 140;
+  pointer-events: none;
+  top: 0;
+  left: 0;
+}
+
+.popover {
+  @include mixins.interactive-focus;
+
+  background-color: var(--pharos-color-ui-10);
+  position: relative;
+  margin: 0;
+  list-style-type: none;
```

Comment:
Can you say a bit about this being here? Feels a bit out of left field. We have it in a couple of other components, but those are very list-oriented.